recipes-private/nsls2ptycho/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set token = environ.get('GH_TOKEN') %}
+
+package:
+  name: nsls2ptycho
+  version: "1.1.1"
+
+source:
+  git_rev: v1.1.1
+  git_url: https://{{ token }}@github.com/leofang/ptycho_gui.git
+
+build:
+  number: 1
+  skip: True # [not py36 or win]
+
+requirements:
+  build:
+    - python 
+    - pip
+    - setuptools
+    - numpy
+  run:
+    - python
+    - fftw >=3
+    - pyfftw
+    - openmpi
+    - mpi4py 
+    - qt >=5
+    - pyqt >=5
+    - numpy
+    - scipy
+    - matplotlib
+    - pillow
+    - h5py
+    - posix_ipc
+    - databroker
+    - attrs >=17.4.0 # needed because anaconda messed up the pytest dependency...see pytest-dev/pytest/pull/3228
+    - numba >=0.41.0
```

Comment:
We tend to sort the dependencies alphabetically. Why not to do it here? :)

Reply:
Done.